src/input_utils.py

```diff
@@ -0,0 +1,87 @@
+import attr
+import tensorflow as tf
+import tensorflow_text as tf_text
+
+from typing import List
```

Comment:
Nit: standard library imports should appear in a block before the third-party imports above.  (https://google.github.io/styleguide/pyguide.html#313-imports-formatting)

Reply:
Thanks, I will be more careful next time!